trio/_tests/test_threads.py

```diff
@@ -463,7 +463,7 @@ async def run_thread(event):
         async with _core.open_nursery() as nursery:
             print("spawning")
             events = []
-            for i in range(COUNT):
+            for _i in range(COUNT):
```

Comment:
to differentiate from B007-induced underscore I think you should use the variable name `_`, I think there were a few other instances earlier as well.

Reply:
Changed in https://github.com/python-trio/trio/commit/11479806a721b0e27e6d4099eab91f59db247976